WBooks/MainMenu/MainMenuController.swift

```diff
@@ -0,0 +1,76 @@
+//
+//  MainMenuController.swift
+//  WBooks
+//
+//  Created by Megan Cole on 14/01/2020.
+//  Copyright © 2020 Wolox. All rights reserved.
+//
+
+import UIKit
+import WolmoCore
+
+class MainMenuController: UIViewController, UITableViewDelegate, UITableViewDataSource {
+    private let _view: MainMenuView = MainMenuView.loadFromNib()!
+    var booksArray: [Book] = []
+    
+    func tableView(_ booksTable: UITableView, numberOfRowsInSection section: Int) -> Int {
+        booksArray.count
+    }
+
+    func tableView(_ booksTable: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
+        let cell = _view.booksTable.dequeue(cell: BookCell.self)!
+        let book = booksArray[indexPath.row]
+
+        cell.bookTitle.text = book.title
+        cell.bookAuthor.text = book.author
+        cell.bookCover.image = book.cover
+
+        return cell
+    }
+    
+    func tableView(_ booksTable: UITableView, heightForHeaderInSection section: Int) -> CGFloat {
+        return 10
+    }
+    
+    required public init?(coder aDecoder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+
+    public override init(nibName nibNameOrNil: String?, bundle nibBundleOrNil: Bundle?) {
+        fatalError("init(nibName:bundle:) has not been implemented")
+    }
+
+    init() {
+        super.init(nibName: .none, bundle: .none)
+    }
+
+    override func loadView() {
+        view = _view
+    }
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        configureBooksTable()
+    }
+    
+    private func configureBooksTable() {
+        booksArray = createBooksArray()
+        _view.booksTable.delegate = self
+        _view.booksTable.dataSource = self
+        _view.booksTable.register(cell: BookCell.self)
+        _view.booksTable.rowHeight = UITableView.automaticDimension
```

Comment:
You can omit the first part of the enum if you want.
```Swift
_view.booksTable.rowHeight = .automaticDimension
```

Reply:
If I omit the `UITableView` it will result in a `Type 'CGFloat' has no member 'automaticDimension'` error.